sync/rpc/rpc.proto

```diff
@@ -10,6 +10,7 @@ service SyncAgentService {
     rpc BackupRemove(BackupRemoveRequest) returns (Empty) {}
     rpc BackupRestore(BackupRestoreRequest) returns (Empty) {}
     rpc BackupRestoreIncrementally(BackupRestoreIncrementallyRequest) returns (Empty) {}
+    rpc BackupFetchProgress(BackupProgressRequest) returns (BackupProgressReply) {}
```

Comment:
`BackupGetStatus`

And separate the GRPC generation code into another commit. Stub the function with `not implemented`. See #364